### PR TITLE
DOC-2090: Document options for setting the inline dialog position

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2090: Document options for setting the inline dialog position

Changes:
1. Edits to introductory text.
2. Addition of a Dialog position section.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/dialog-configuration.adoc [hunk 1]
```diff
@@ -62,3 +67,47 @@ For more information on the `+dialogApi+` object that is passed to some of the c
...
+    ]
+  },
+  buttons: [ // A list of footer buttons
+    {
+      type: 'submit',
+      text: 'OK'
+    }
+  ]
+  { inline: 'toolbar' }
+});
```
Comment (kemister85) [inline modules/ROOT/pages/dialog-configuration.adoc:112]:
```suggestion
  buttons: [ // A list of footer buttons
    {
      type: 'submit',
      text: 'OK'
    }
  ]
}, { inline: 'toolbar' });
```

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/pages/dialog-configuration.adoc [hunk 2]
```diff
@@ -62,3 +67,47 @@ For more information on the `+dialogApi+` object that is passed to some of the c
...
 
 Any callback function that is not passed a `+details+` object assumes that the dialog will only contain one component which can trigger it or that it does not matter if the function is triggered by multiple components. For example, `+onSubmit+` is only triggered when a user clicks on a *Submit* type footer button, and {productname} assumes that a dialog will only have one *Submit* type button. In comparison, `+onCancel+` and `+onClose+` are both triggered by clicking the `+X+` button in the top right of a dialog or by clicking a *Cancel* type footer button. These two buttons have the same functionality, and therefore {productname} does not differentiate between them.
+
+== Dialog position
+
+By default a dialog presents as a modal object in the center of the editor viewport.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/dialog-configuration.adoc:73]:
```suggestion
By default a dialog is shown as a modal in the center of the editor viewport.
```

File: modules/ROOT/pages/dialog-configuration.adoc [hunk 3]
```diff
@@ -62,3 +67,47 @@ For more information on the `+dialogApi+` object that is passed to some of the c
...
+
+This presented position can be changed for a given dialog by adding an `inline` value to the dialog’s configuration.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/dialog-configuration.adoc:75]: Actually, `{inline: 'bottom'}` should be provided in the second argument to windowManager.open() as seen in the example below

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/dialog-configuration.adoc (hunk 3 above)
Comment (brian-forte) [reply modules/ROOT/pages/dialog-configuration.adoc:75]: @tiny-ben-tran,

> Actually, `{inline: 'bottom'}` should be provided in the second argument to windowManager.open() as seen in the example below

¿que?

If, by _example below_, you mean the example I added, said example doesn’t have an `{inline: 'bottom'}` value in it at all.

What am I missing?

Comment (tiny-ben-tran) [reply modules/ROOT/pages/dialog-configuration.adoc:75]: Sorry. I meant `inline: 'toolbar'`.

Comment (brian-forte) [reply modules/ROOT/pages/dialog-configuration.adoc]: Appropriate edits done.

File: modules/ROOT/pages/dialog-configuration.adoc [hunk 4]
```diff
@@ -62,3 +67,46 @@ For more information on the `+dialogApi+` object that is passed to some of the c
...
+== Dialog position
+
+By default a dialog is shown as a modal in the center of the editor viewport.
+
+This presented position can be changed by providing a second argument, with an appropriate value, to `editor.windowManager.open()`.
+
+[[inline]]
+=== `+inline+`
+
+In a dialog configuration, the `+inline+` option sets the dialog’s presented position to one of two possiblities.
```
Comment (shanmen-tiny) [inline modules/ROOT/pages/dialog-configuration.adoc:80]: The configuration for inline dialog is passed as a second argument to the `windowManager.open`, it's not part of the `dialog` configuration, so should be in the second argument. Maybe:

```suggestion
The `+inline+` option passed to the `editor.windowManager.open` sets the dialog’s presented position to one of two possiblities.
```

**Review by shanmen-tiny — APPROVED** [review]

**Pull request merged**